R/printnum.R

```diff
@@ -302,6 +345,111 @@ printnum.papaja_labelled <-function(x, ...){
 }
 
 
+#' Prepare numeric values for printing as p value
+#'
+#' Convenience wrapper for \code{\link{printnum}} to print \emph{p} values.
+#'
+#' @param x Numeric. The \emph{p} value(s) to report.
+#' @param digits Integer. The desired number of digits after the decimal point, passed on to \code{\link{formatC}}.
+#' @inheritParams printnum.numeric
+#' @examples
+#' printp(0.05)
+#' printp(0.0005)
+#' printp(0.99999999)
+#' printp(c(.001, 0), add_equals = TRUE)
+#' @export
+
+printp <- function(x, digits = 3L, na_string = "", add_equals = FALSE) {
+  validate(x, check_class = "numeric", check_range = c(0, 1), check_NA = FALSE)
+  validate(na_string, check_class = "character")
```

Comment:
I think the `na_string`- and `add_equals`-validation here is redundant with the checking done in `printnum()`, right?

Reply:
Yes. It would be my pleasure and privilege to remove these extraneous validation steps.  :laughing:

Reply:
Done.